tests/api-watch/test_diff_modtpe_decl.ml

```diff
@@ -0,0 +1,75 @@
+open Api_watch
+open Test_helpers
+
+let%expect_test "Module_type with value changes" =
```

Comment:
The test name does not correspond to what we are testing here. If you want to test for the removal of a value, note that you can define an empty module type:
```ocaml
module type T = sig
end
```

If you wish to test a module type going from concrete to abstract, then the test name should reflect that.

Reply:
Thank you